src/zk.py

```diff
@@ -106,6 +101,19 @@ def __init__(self, config, plugins):
             for line in traceback.format_exc().split('\n'):
                 logging.error(line.rstrip())
 
+
+    def get_lock_contender_name(self):
```

Comment:
We can calculate `self._lock_contender_name` during class initialization only once, and get rid of this method, using just `self._lock_contender_name` property when needed.

Reply:
hostname may be incorrect when starting the VM, so I prefer to call the hostname() function every time.